plugwise_usb/helpers/cache.py

```
@@ -0,0 +1,160 @@
+
+    async def initialize_cache(self, create_root_folder: bool = False) -> None:
+        """Set (and create) the plugwise cache directory to store cache file."""
+        if self._root_dir != "":
+            if not create_root_folder and not await ospath.exists(self._root_dir):
+                raise CacheError(f"Unable to initialize caching. Cache folder '{self._root_dir}' does not exists.")
+            cache_dir = self._root_dir
+        else:
+            cache_dir = self._get_writable_os_dir()
+        await makedirs(cache_dir, exist_ok=True)
+        self._cache_path = cache_dir
+        
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)